### PR TITLE
Create Syslogins Catalog View in Babelfish

### PR DESCRIPTION
**Pull request by deepakshi-mittal**
Create Syslogins Catalog View in Babelfish
Task: BABEL-2476 

### Description
Create Syslogins Catalog View in Babelfish

### Issues Resolved
Syslogins catalog added to Babelfish

### Test Scenarios Covered ###
* **Use case based -**
- Tested Syslogins view using two logins

### Check List
- [ ] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is under the terms of the Apache 2.0 and PostgreSQL licenses, and grant any person obtaining a copy of the contribution permission to relicense all or a portion of my contribution to the PostgreSQL License solely to contribute all or a portion of my contribution to the PostgreSQL open source project.

For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/babelfish-for-postgresql/babelfish_extensions/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: contrib/babelfishpg_tsql/sql/ownership.sql [hunk 1]
```diff
@@ -304,6 +304,61 @@ FROM pg_catalog.pg_roles AS Base INNER JOIN sys.babelfish_authid_login_ext AS Ex
...
+Base.modify_date AS updatedate,
+Base.create_date AS accdate,
+CAST(0 AS INT) AS totcpu,
+CAST(0 AS INT) AS totio,
+CAST(0 AS INT) AS spacelimit,
+CAST(0 AS INT) AS timelimit,
+CAST(0 AS INT) AS resultlimit,
+Base.name AS name,
+Base.default_database_name AS dbname,
+Base.default_language_name AS default_language_name,
```
Comment (HarshLunagariya) [inline contrib/babelfishpg_tsql/sql/ownership.sql:322]: Please add CAST() here even if the datatype are same from the Base view.

File: contrib/babelfishpg_tsql/sql/ownership.sql [hunk 2]
```diff
@@ -304,6 +304,61 @@ FROM pg_catalog.pg_roles AS Base INNER JOIN sys.babelfish_authid_login_ext AS Ex
...
+CAST(Base.name AS SYS.NVARCHAR(128)) AS loginname,
+CAST(NULL AS SYS.NVARCHAR(128)) AS password,
+CAST(0 AS INT) AS denylogin,
+CAST(1 AS INT) AS hasaccess,
```
Comment (HarshLunagariya) [inline contrib/babelfishpg_tsql/sql/ownership.sql:326]: Does `hasaccess` depend on whether login is enabled or disabled

**Review by HarshLunagariya — CHANGES_REQUESTED** [review]

File: contrib/babelfishpg_tsql/sql/ownership.sql (hunk 2 above)
Comment (deepakshi-mittal) [reply contrib/babelfishpg_tsql/sql/ownership.sql:326]: No its always 1. There is no way right now we can disable access.

File: contrib/babelfishpg_tsql/sql/ownership.sql (hunk 1 above)
Comment (deepakshi-mittal) [reply contrib/babelfishpg_tsql/sql/ownership.sql:322]: in earlier reviews, I was asked to remove any recasting if base View has same datatypes

**Review by xhfanhe — APPROVED** [review]

**Pull request merged**